### PR TITLE
Django 1.11 support, code cleanup and documentation clarification

### PR DESCRIPTION
**Pull request by jouvin**
- `README.md` with working instructions for installing SLAM as a UWSGI application
- `slam` directory renamed `src`
- Import of uncommited changes at LAL
- Add `conf` directory, at the same level as `src`, as the first location looked for `configure.py`
- Deployment script refactoring and cleanup
- Django 1.11 support (and requirement)

### CODE REVIEW

**Comment by jouvin** [comment]
Ready for merging...

**Comment by jouvin** [comment]
Really ready to merge!

**Comment by jouvin** [comment]
Every pending issue has been fixed and the code cleanup planned is complete.

**Comment by jouvin** [comment]
Issue found with object modification... Fix coming soon!

**Comment by jouvin** [comment]
Issue related to an [attribute name change](https://stackoverflow.com/questions/22368190/django-cant-access-raw-post-data) has been fixed. The current version of this PR is now in prod at LAL.

File: README.md [hunk 1]
```diff
@@ -8,122 +7,247 @@ various other computer and network management software and frameworks like
...
+    ```
+
+
+## SLAM CLI Usage
+
+*Note: if you are using the recommended virtualenv-based configuration, you must activate the virtual environment before running `slam-cli`. This is
+typically done with:*
+
+```bash
+. /opt/slam/venv/bin/activate
```
Comment (aflp91) [inline README.md]: Probablement : `. /opt/slam/.venv/bin/activate` (l'environnement virtuel a été défini quelques lignes plus haut dans `.venv`)

File: README.md [hunk 2]
```diff
@@ -8,122 +7,247 @@ various other computer and network management software and frameworks like
...
+a `conf` directory at the same level as `src` directory or in `/etc/slam`. A template 
+exists in `conf` directory.
+
+Configuring services like DNS and DHCP with information from SLAM DB requires a script
+typically placed in `scripts` directory (at the same level as `src` directory). The LAL
+script provided in the repository can be used as a template. The name of the script must be
+specified as the value of `RELOAD_SCRIPT` in `configure.py`.
+
+Additionally, if you reuse the LAL deployment script, it requires a `run` directory at the same level as
+the `scripts` directory. This directory must be owned by writable by the USWGI account used
```
Comment (aflp91) [inline README.md]: _s/owned by writable/owned and writable/_ ?

**Pull request merged**